src/a4/domain/Street.java

```diff
@@ -57,12 +60,15 @@ public void addToNeighborhood(Neighborhood n) {
 		neighborhood = n;
 	}
 
+	@Override
 	public int getRent() {
-		int rent = value;
-		if (0 == houseCount && 0 == hotelCount && owner == neighborhood.belongsTo()) {
-			rent = rent * 2;
-		}
-		return value;
-		// TODO: this needs unit tests still
+		if (houseCount > 0)
+			return rent[houseCount];
+		else if (hotelCount > 0)
+			return rent[hotelCount * 5];
+		else if (houseCount == 0 && hotelCount == 0 && neighborhood.belongsTo() != null && neighborhood.belongsTo() == owner)
```

Comment:
Should we use the player equals method? will using == to compare the owner work?